openedx/core/djangoapps/discussions/apps.py

```diff
@@ -15,9 +15,15 @@ class DiscussionsConfig(AppConfig):
     name = 'openedx.core.djangoapps.discussions'
     plugin_app = {
         PluginURLs.CONFIG: {
+            # TODO: Remove the LMS path once its usage has been removed from frontend-app-course-authoring.
```

Comment:
I am missing some context can you explain which URL is this? is that API URL for discussions settings?